README.md

```diff
@@ -137,9 +137,9 @@ The default settings don't conflict, and Prettier plugins can quickly fix up ord
 | [restrict-dependency-ranges](docs/rules/restrict-dependency-ranges.md) | Restricts the range of dependencies to allow or disallow specific types of ranges.                |      |    | 💡 |    |
 | [sort-collections](docs/rules/sort-collections.md)                     | Dependencies, scripts, and configuration values must be declared in alphabetical order.           | ✔️ ✅ | 🔧 |    |    |
 | [unique-dependencies](docs/rules/unique-dependencies.md)               | Checks a dependency isn't specified more than once (i.e. in `dependencies` and `devDependencies`) | ✔️ ✅ |    | 💡 |    |
-| [valid-author](docs/rules/valid-author.md)                             | Enforce that the author field is a valid npm author specification                                 | ✔️ ✅ |    |    |    |
+| [valid-author](docs/rules/valid-author.md)                             | Enforce that the `author` property is valid.                                                      | ✔️ ✅ |    |    |    |
```

Comment:
😕 [Bug] This is a loss of functionality IMO. The current/previous description is a little more informative (though the backticks around the name are definitely an improvement). How about allowing an optional description for the utility function?

Reply:
Oh, the original version doesn't seem clear at all to me, and across all the valid- rules, we had slightly different versions of that text (I blame me for that 😓!).   I guess, why would we want to have that be different for any of them?  Happy to change the text to something you think is more descriptive, but I think we'd want them to all be the same, right?